internal/npo-ext.owl

```diff
@@ -3185,7 +3197,50 @@
         <rdfs:subClassOf rdf:resource="http://purl.bioontology.org/ontology/npo#NPO_1999"/>
         <rdfs:label xml:lang="en">disc centrifuge</rdfs:label>
     </owl:Class>
+    
+    <!-- http://purl.enanomapper.org/onto/ENM_0000104 -->
+        
+    <owl:Class rdf:about="http://purl.enanomapper.org/onto/ENM_0000104">
+        <rdfs:subClassOf rdf:resource="http://purl.bioontology.org/ontology/npo#NPO_1430"/>
+        <rdfs:label xml:lang="en">Full field transmission X-ray microscopy</rdfs:label>
```

Comment:
Shouldn't  `Full field transmission X-ray microscopy`  be a subclass of X-ray microscopy, instead of `transmission electron microscopy` NPO_1430 ?  https://www.sciencedirect.com/topics/chemistry/x-ray-microscopy

Reply:
Yes it should..